config/settings_schema.json

```diff
@@ -151,74 +151,44 @@
         ],
         "default": "1600",
         "label": "t:settings_schema.layout.settings.page_width.label"
-      },
-      {
-        "type": "header",
-        "content": "Desktop"
-      },
+        },
       {
         "type": "range",
-        "id": "spacing_sections_desktop",
+        "id": "spacing_sections",
         "min": 0,
         "max": 120,
         "step": 4,
         "unit": "px",
-        "label": "Vertical space between sections",
+        "label": "t:settings_schema.layout.settings.spacing_sections.label",
         "default": 0
       },
-      {
-        "type": "range",
-        "id": "spacing_grid_horizontal_desktop",
-        "min": 4,
-        "max": 72,
-        "step": 4,
-        "default": 8,
-        "unit": "px",
-        "label": "Multicolumn horizontal space"
-      },
-      {
-        "type": "range",
-        "id": "spacing_grid_vertical_desktop",
-        "min": 4,
-        "max": 72,
-        "step": 4,
-        "default": 8,
-        "unit": "px",
-        "label": "Multicolumn vertical space"
-      },    
       {
         "type": "header",
-        "content": "Mobile"
+        "content": "t:settings_schema.layout.settings.header__grid.content"
       },
       {
-        "type": "range",
-        "id": "spacing_sections_mobile",
-        "min": 0,
-        "max": 120,
-        "step": 4,
-        "unit": "px",
-        "label": "Vertical space between sections",
-        "default": 52
+        "type": "paragraph",
+        "content": "t:settings_schema.layout.settings.paragraph__grid.content"
       },
       {
         "type": "range",
-        "id": "spacing_grid_horizontal_mobile",
+        "id": "spacing_grid_horizontal",
         "min": 4,
-        "max": 40,
+        "max": 72,
```

Comment:
this can definitely mess with things: 
- sliders [video](https://screenshot.click/21-12-krso0-zsxn7.mp4)
- collection cards in a collage scenario [screenshot](https://screenshot.click/21-12-l0q6u-mywuu.png)
- collection cards in a collection list section [screenshot](https://screenshot.click/21-12-s9ok4-mzwe0.png)

Not that those should be fixed here but maybe we can limit the max value to something different. Otherwise we will need follow up issue to tackle those scenarios. 

I also noticed a couple places that will also need addressing, noting them here so I can come back to it when creating issues later 😅 : 
- those spacing value mess with the product page because the `grid__item` have padding override: [screenshot](https://screenshot.click/21-12-se48o-fjxlv.png) (in a stacked and thumbnail layout)
- looks like the spacing is gone under the heading on a collection page. So filters look very close to it, [screenshot](https://screenshot.click/21-12-2tgie-c3ds2.png)
- feat. product slider doesn't seem to be using the right spacing values on a tablet sized viewport: [screenshot](https://screenshot.click/21-12-zht23-z119o.png)

Reply:
Thanks Ludo! There is a lot of great point we can discuss:

>  Not that those should be fixed here but maybe we can limit the max value to something different

- Do we have an idea of what could be a max value that could not mess with the described findings? We can decrease the  max and take more time to increase it later, but I am curious if regardless we would need to adjust all the instances. 🤔

> those spacing value mess with the product page because the grid__item have padding override

- [ ]  I  would  expect this being a quick  follow up as it means  the setting is not applied consistently

> looks like the spacing is gone under the heading on a collection page. So filters look very close to it

- This should be solved by this section padding follow up issue https://github.com/Shopify/dawn/issues/1039

> feat. product slider doesn't seem to be using the right spacing values on a tablet sized viewport

-  I believe this was fixed already, can't find when 🤔 but otherwise let's open an issue please! 🙌